content/en/docs/tutorials/security/_index.md

```diff
@@ -3,3 +3,4 @@ title: "Security"
 weight: 40
 ---
 
+Kubernetes security is important for most deployments and production facilities. Below are some initial suggestions for configuring the security options at various levels. The Kubernetes project recommends looking into these suggestions before running a production-based cluster.
```

Comment:
```suggestion
Kubernetes security is important for most deployments and production facilities. Below are some suggestions for configuring the security options at various levels. The Kubernetes project recommends looking into these suggestions before running a production-based cluster.
```
Could you explain why did you use _initial_ here and in the next sentence, avoiding it? I don't see any context of it.

Reply:
@kramse [suggested](https://github.com/kubernetes/website/issues/39249) such wording.

Reply:
> ```suggestion
> Kubernetes security is important for most deployments and production facilities. Below are some suggestions for configuring the security options at various levels. The Kubernetes project recommends looking into these suggestions before running a production-based cluster.
> ```
> Could you explain why did you use _initial_ here and in the next sentence, avoiding it? I don't see any context of it.

But I think that putting the initial in the second line as well would not make sense. Also it is relevant in the first line because we indeed are making some initial suggestions and the user can go for more advanced solutions. What do you think??? 🤔

Reply:
@sftim , @NitishKumar06 ??

Reply:
agreed @nate-double-u